src/main/resources/messages/LibertyBundles.properties

```diff
@@ -41,6 +41,8 @@ liberty.dev.not.started.notification.content=Liberty dev mode has not been start
 liberty.project.does.not.resolve=Unable to {0}: Could not resolve project. Ensure you run the Liberty action from the Liberty tool window.
 liberty.build.file.does.not.resolve=Unable to {0}: Could not resolve build file for {1}. Ensure you run the Liberty action from the Liberty tool window.
 liberty.action.cannot.start=Liberty action was not able to start
+liberty.project.file.selection.dialog.title=Liberty project file
+liberty.project.file.selection.dialog.message=Select Liberty project file for {0}
```

Comment:
Rather than Liberty project file, should be "Liberty project" and "Select Liberty project for {0}"

Reply:
I've updated the PR with your suggested change to the messages.